docs/inference.md

```diff
@@ -41,6 +41,29 @@ model.resize_token_embeddings(model.config.vocab_size + 1)
 ```
 Padding would be required for batch inference. In this this [example](../inference/inference.py), batch size = 1 so essentially padding is not required. However,We added the code pointer as an example in case of batch inference.
 
+**Code Llama**
```

Comment:
Move this section after the Chat models section at the end

Reply:
@HamidShojanazeri Will be better to move this Note right next to the Infilling example to remove confusion.